### ISSUE TITLE
rpm_packages table broken on Fedora 33 due to new RPM database format

### ISSUE THREAD

**Issue by gkleiman** [osquery/osquery#6895]
# Bug report

The `rpm_packages` table doesn't work on Fedora 33. The problem seems to be caused by Fedora having changed the format of the RPM database from Berkeley DB to a new Sqlite format ([details](https://fedoraproject.org/wiki/Changes/Sqlite_Rpmdb)).

According to Fedora's doc, support for the new format is introduced in `librpm` 4.16, so osquery's librpm dep will have to be bumped. It looks like someone is working on a related PR: https://github.com/osquery/osquery/pull/6850

### What operating system and version are you using?

```
 version = Fedora release 33 (Thirty Three)
   build =
platform = rhel
```

### What version of osquery are you using?

```
version = 4.6.0
```

### What steps did you take to reproduce the issue?
Ran `select * from rpm_packages` on a `osqueryi --verbose` session as root:

```
osquery> select * from rpm_packages;
I0111 18:05:16.341981   839 glog_logger.cpp:49] RPM notice: Failed to read auxiliary vector, /proc not mounted?
I0111 18:05:16.343933   839 glog_logger.cpp:49] RPM notice: cannot open Packages index using db5 - Permission denied (13)
I0111 18:05:16.344034   839 glog_logger.cpp:49] RPM notice: cannot open Packages database in
I0111 18:05:16.344218   839 glog_logger.cpp:49] RPM notice: cannot open Packages index using db5 - Permission denied (13)
I0111 18:05:16.344285   839 glog_logger.cpp:49] RPM notice: cannot open Packages database in
osquery>
```

### What did you expect to see?
A list of the rpm packages installed on the host.

### What did you see instead?
RPM notices about errors trying to open the Packages database.

**Comment by theopolis** [comment]
I am working towards implementing RPMDB SQLite support, the next step is  https://github.com/osquery/osquery/pull/6931 and then the final step (the last PR) will be adding a test and configuration for SQLite.

**Comment by directionless** [comment]
Is this fixed by #6939

**Comment by theopolis** [comment]
Yes :)
